version.py

```diff
@@ -6,5 +6,5 @@
 # If you need to change the version, do so in the project.toml, e.g. by using `poetry version X.Y.Z`.
 MAJOR = 0
 MINOR = 2
```

Comment:
I think, we should go up with minor, here because we added to the interface

Reply:
It's still backward-compatible.